app/components/InstallExtensionButton.jsx

```diff
@@ -26,6 +26,11 @@ const BrowserIcons = {
 const DefaultLink = "https://github.com/getAlby/lightning-browser-extension";
 
 function Install({ style }) {
+
+  function trackDownload() {
+    plausible('Download');
```

Comment:
this is not defined here because of the remix context. need to see how to define a `window.plausible` function.

Reply:
here's an example where they detect whether they're on the server and set an empty function as a fallback:
https://remix.run/docs/en/v1/guides/constraints#uselayouteffect

Reply:
This example also looks relevant, they fix a `localstorage` call (which has the same problem as `window` on the server): https://remix.run/docs/en/v1/guides/constraints#rendering-with-browser-only-apis